web/beacon-app/src/features/topics/components/QueryForm.tsx

```diff
@@ -0,0 +1,37 @@
+import { Trans } from '@lingui/macro';
+import { Button, TextField } from '@rotational/beacon-core';
+
+type QueryFormProps = {
+  defaultEnSQL: string;
+  queryHandler: (query: string) => void;
+};
+
+const QueryForm = ({ defaultEnSQL, queryHandler }: QueryFormProps) => {
+  const onQuery = (e: any) => {
+    e.preventDefault();
+    queryHandler(e.target.value);
+  };
+
+  return (
+    <div className="mt-4 flex space-x-2">
+      <TextField
+        labelClassName="font-semibold"
```

Comment:
There isn't a label for this input. Did you meant to make the value semibold?

Reply:
I tried making the value semi-bold, but I remember the design system doesn't provide this ability. To interact with the input style, I will add inputClassname while working on the Tag component.